waku/waku_core/message/digest.nim

```diff
@@ -22,6 +22,15 @@ type WakuMessageDigest* = array[32, byte]
 converter toBytesArray*(digest: MDigest[256]): WakuMessageDigest =
   digest.data
 
+converter toBytesFromInt64*(num: int64): seq[byte] =
```

Comment:
I think there is already a utility module to achieve this:

https://github.com/status-im/nim-stew/blob/2c2544aec13536304438be045bfdd22452741466/stew/endians2.nim#L93-L105